File: andrescastro-javasenior-frontend/src/app/models/new-student.ts
export class NewStudent {
    id?: number;
    rut: string;
    name: string;
    lastName: string;
    age: number;
    nameUser: string;
    password: string;
    courses: [];
    constructor(rut: string, name: string, lastName: string , age: number, nameUser: string , password: string) {
        this.rut = rut;
        this.name = name;
        this.lastName = lastName;
        this.age = age;
        this.nameUser = nameUser;
        this.password = password;
    }
}

File: andrescastro-javasenior-backend/src/main/java/javasenior/backend/security/dto/NewStudent.java
package javasenior.backend.security.dto;
import javax.validation.constraints.Min;
import javax.validation.constraints.NotBlank;
import javax.validation.constraints.NotNull;


public class NewStudent {

    @NotBlank
    private String rut;
    @NotBlank
    private String name;
    @NotBlank
    private String lastName;
    @NotNull
    @Min(value = 18, message = "Age should not be less than 18")
    private int age;
    @NotBlank
    private String nameUser;

    @NotBlank
    private String password;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getRut() {
        return rut;
    }

    public void setRut(String rut) {
        this.rut = rut;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public String getNameUser() {
        return nameUser;
    }

    public void setNameUser(String nameUser) {
        this.nameUser = nameUser;
    }



    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

}
